setup.py

```diff
@@ -18,7 +18,7 @@
     "attrs",
     "pyyaml~=6.0",
     "numexpr~=2.0",
-    "numpy~=1.20",
+    "numpy~=2.0",
```

Comment:
Since `numpy` 2.0 is a very new release (June 16, 2024), it would be helpful for me (and I suspect many other users) to not yet require version 2.